cmd/commands/service/command.go

```diff
@@ -69,10 +78,20 @@ func NewCommand() *cli.Command {
 		Flags: []cli.Flag{
 			identityFlag, identityPassphraseFlag,
 			openvpnProtocolFlag, openvpnPortFlag,
+			agreedTermsConditionsFlag,
 		},
 		Action: func(ctx *cli.Context) error {
+			if !ctx.Bool(agreedTermsConditionsFlag.Name) {
+				fmt.Println(metadata.VersionAsSummary(metadata.LicenseCopyright(
+					"run program with '--license.warranty' option",
```

Comment:
should be `run program with 'myst {licenseCommand.name} {warantFlag.name}'` and same with conditions

Reply:
fixed